frontend/public/locales/en/create.json

```diff
@@ -255,6 +255,8 @@
     "creation.ocp.otp.instance.flavor": "Instance flavor",
     "creation.ocp.cluster.ost.external.network.name": "External network name",
     "creation.ocp.cluster.ost.must.enter.external.network.name": "External network name is required.",
+    "creation.ocp.cluster.ost.lb.floating.ip": "LB floating IP",
+    "placeholder.creation.ocp.cluster.ost.lb.floating.ip": "Enter LB floating IP",
```

Comment:
On main branch, `create.json` file is obsolete. Please add these in `plugin__acm-plugin.json`